crates/wasi-common/src/sys/windows/fd.rs

```diff
@@ -34,27 +35,29 @@ pub(crate) fn fdstat_get(fd: &File) -> Result<types::Fdflags> {
     Ok(fdflags)
 }
 
-pub(crate) fn fdstat_set_flags(fd: &File, fdflags: types::Fdflags) -> Result<Option<OsHandle>> {
-    let handle = fd.as_raw_handle();
-
+// TODO is this correct if we're modifying Stdio on Windows? If setting flags
```

Comment:
Oh good find, thanks! So if I understood it right, modifying stdio should error out on Windows since the handle was obtained from `GetStdHandle` and not `CreateFile`. I'm keen to leave some sort of TODO here anyway to investigate this further on Windows.